Sources/FutureType.swift

```diff
@@ -30,14 +30,14 @@ public protocol FutureType: CustomDebugStringConvertible, CustomReflectable {
     /// A type that represents the result of some asynchronous operation.
     associatedtype Value
 
-    /// Call some function once the value is determined.
+    /// Call some function `body` once the value is determined.
```

Comment:
The stdlib docs seem to be coalescing around "the `body` closure" (following the new internal parameter rule). I'll see what I can do.